### ISSUE TITLE
 'caption': 'ssss'

### ISSUE THREAD

**Issue by daitranskku** [callsys/DynRefer#9]
Hi, 
Thank you for providing this public code. I tried to rerun the DynRefer code and received this output as a region-live captioning.
However, the 'caption' shows 'sss', even though other attributes can be extracted.

**{'score': nan, 'caption': 'ssss', 'attributes': ['gender:male/man/guy/boy', 'maturity:adult..]**

Does this problem come from code?
Thank you.

**Comment by callsys** [comment]
Hello, are you using a retrained model or the models provided by us. Based on the results you've provided, it seems that the decoder head of the language model outputs "nan", which is abnormal.

**Comment by daitranskku** [comment]
Hi, I just used the pretrained model. But other attributes can be extracted.

**Comment by daitranskku** [comment]
Here is the result from dynrefer_inference in demo using "refcocog_ft.pth"
{'score': nan, 'caption': 'ssss', 'attributes': ['color:white', 'face expression:sleepy/sleeping', 'position:horizontal/lying', 'state:folded/bend', 'state:on', 'texture:soft/fluffy/furry/hairy', 'tone:light/bright'], 'class': 'dog'}

**Comment by callsys** [comment]
Hello, on which dataset or images did you get this result.

**Comment by daitranskku** [comment]
Hi, from dynrefer_inference.py 
image_path = "demo/examples/COCO_val2014_000000000074.jpg"

**Comment by callsys** [comment]
I just test the image with bbox [64, 255, 383, 382] using "refcocog_ft.pth". The result is "{'score': 0.05080917105078697, 'caption': 'the white dog', 'attributes': ['color:white', 'face expression:sleepy/sleeping', 'position:horizontal/lying', 'state:folded/bend', 'state:on', 'texture:soft/fluffy/furry/hairy', 'tone:light/bright'], 'class': 'dog'}". It seems there are differences between our environment.

**Comment by daitranskku** [comment]
Thank you for your support. I will reinstall the environment.

**Comment by callsys** [comment]
accelerate==0.27.2
altair==5.2.0
annotated-types==0.6.0
antlr4-python3-runtime==4.9.3
asttokens==2.4.1
attrs==23.2.0
backcall==0.2.0
backports.zoneinfo==0.2.1
bleach==6.1.0
blinker==1.7.0
blis==0.7.11
braceexpand==0.1.7
cachetools==5.3.2
catalogue==2.0.10
certifi==2023.11.17
cfgv==3.4.0
charset-normalizer==3.3.2
click==8.1.7
cloudpathlib==0.16.0
confection==0.1.4
contexttimer==0.3.3
contourpy==1.1.1
cycler==0.12.1
cymem==2.0.8
DCNv4==1.0.0.post2
decorator==5.1.1
decord==0.6.0
deepspeed==0.13.4
distlib==0.3.8
einops==0.7.0
en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.7.1/en_core_web_sm-3.7.1-py3-none-any.whl#sha256=86cc141f63942d4b2c5fcee06630fd6f904788d2f0ab005cce45aadb8fb73889
executing==2.0.1
fairscale==0.4.4
filelock==3.13.1
fonttools==4.47.0
fsspec==2023.12.2
ftfy==6.1.3
gitdb==4.0.11
GitPython==3.1.40
hjson==3.1.0
huggingface-hub==0.20.2
identify==2.5.33
idna==3.6
imageio==2.33.1
importlib-metadata==6.11.0
importlib-resources==6.1.1
iopath==0.1.10
ipython==8.12.3
jedi==0.19.1
Jinja2==3.1.2
joblib==1.3.2
jsonschema==4.20.0
jsonschema-specifications==2023.12.1
kaggle==1.6.0
kiwisolver==1.4.5
langcodes==3.3.0
lazy_loader==0.3
markdown-it-py==3.0.0
MarkupSafe==2.1.3
matplotlib==3.7.4
matplotlib-inline==0.1.6
mdurl==0.1.2
mpmath==1.3.0
murmurhash==1.0.10
networkx==3.1
ninja==1.11.1.1
nltk==3.8.1
nodeenv==1.8.0
numpy==1.24.4
nvidia-cublas-cu12==12.1.3.1
nvidia-cuda-cupti-cu12==12.1.105
nvidia-cuda-nvrtc-cu12==12.1.105
nvidia-cuda-runtime-cu12==12.1.105
nvidia-cudnn-cu12==8.9.2.26
nvidia-cufft-cu12==11.0.2.54
nvidia-curand-cu12==10.3.2.106
nvidia-cusolver-cu12==11.4.5.107
nvidia-cusparse-cu12==12.1.0.106
nvidia-nccl-cu12==2.18.1
nvidia-nvjitlink-cu12==12.3.101
nvidia-nvtx-cu12==12.1.105
omegaconf==2.3.0
opencv-python==4.9.0.80
opencv-python-headless==4.5.5.64
opendatasets==0.1.22
packaging==23.2
pandas==2.0.3
parso==0.8.3
peft==0.8.2
pexpect==4.9.0
pickleshare==0.7.5
pillow==10.2.0
pkgutil_resolve_name==1.3.10
platformdirs==4.1.0
plotly==5.18.0
portalocker==2.8.2
pre-commit==3.5.0
preshed==3.0.9
prompt-toolkit==3.0.43
protobuf==4.25.1
psutil==5.9.7
ptyprocess==0.7.0
pure-eval==0.2.2
py-cpuinfo==9.0.0
pyarrow==14.0.2
pycocoevalcap==1.2
pycocotools==2.0.7
pydantic==2.5.3
pydantic_core==2.14.6
pydeck==0.8.1b0
Pygments==2.17.2
pynvml==11.5.0
pyparsing==3.1.1
python-dateutil==2.8.2
python-magic==0.4.27
python-slugify==8.0.1
pytz==2023.3.post1
PyWavelets==1.4.1
PyYAML==6.0.1
referencing==0.32.1
regex==2023.12.25
requests==2.31.0
rich==13.7.0
rpds-py==0.16.2
safetensors==0.4.2
salesforce-lavis==1.0.2
SceneGraphParser==0.1.0
scikit-image==0.21.0
scikit-learn==1.3.2
scipy==1.10.1
seaborn==0.13.2
sentencepiece==0.1.99
six==1.16.0
smart-open==6.4.0
smmap==5.0.1
spacy==3.7.2
spacy-legacy==3.0.12
spacy-loggers==1.0.5
srsly==2.4.8
stack-data==0.6.3
streamlit==1.29.0
sympy==1.12
tabulate==0.9.0
tenacity==8.2.3
text-unidecode==1.3
textblob==0.17.1
thinc==8.2.2
threadpoolctl==3.5.0
tifffile==2023.7.10
timm==0.4.12
tokenizers==0.13.3
toml==0.10.2
toolz==0.12.0
torch==2.1.2
torchvision==0.16.2
tornado==6.4
tqdm==4.66.1
traitlets==5.14.1
transformers==4.28.0
triton==2.1.0
typer==0.9.0
typing_extensions==4.9.0
tzdata==2023.4
tzlocal==5.2
urllib3==2.1.0
validators==0.22.0
virtualenv==20.25.0
wasabi==1.1.2
watchdog==3.0.0
wcwidth==0.2.13
weasel==0.3.4
webdataset==0.2.86
webencodings==0.5.1
zipp==3.17.0
This is my environment, I currently cannot help you pinpoint the issue. Please compare it with my environment!
